ApplicationCode/ModelVisualization/RivElementVectorResultPartMgr.h

```diff
@@ -54,16 +54,18 @@ class RivElementVectorResultPartMgr : public cvf::Object
 private:
     struct ElementVectorResultVisualization
     {
-        ElementVectorResultVisualization( cvf::Vec3d faceCenter, cvf::Vec3d faceNormal, double result )
+        ElementVectorResultVisualization( cvf::Vec3d faceCenter, cvf::Vec3d faceNormal, double result, double characteristicCellSize )
```

Comment:
characteristicCellSize is used elsewhere as a "typical cell size", but here you use it as the real cell size?

Reply:
I didn't know it was already used somewhere else with a different meaning. I use it as the cube root of the real cell volume. But I could call it approximatedCellLength to avoid confusion.